include/dsn/tool/latency_tracer.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2017-present, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#pragma once
+#include <iterator>
+#include <string>
+#include <map>
+#include <dsn/service_api_c.h>
+#include <dsn/dist/fmt_logging.h>
+
+namespace dsn {
+namespace tool {
+
+struct latency_tracer
+{
+public:
+    uint64_t id;
+    std::string name;
+    uint64_t start_time;
+    uint64_t last_time;
+    uint64_t end_time;
+    std::string request_type;
+    uint64_t prepare_ack_time_on_secondary;
+    std::set<uint64_t> prepare_time_used_on_primary;
+    std::unordered_map<std::string, uint64_t> points;
```

Comment:
Cloud you please add comments for those varieties? `last_time`, `request_type`, `prepare_ack_time_on_secondary`, `prepare_time_used_on_primary` and `points`. I just can not understand what is the meaning of them.
It seems that `prepare_ack_time_on_secondary` is used only for secondary, `prepare_time_used_on_primary` is used only for primary, I think you should add special comments for them.

Reply:
I think as a tool,  these `special` variable shouln't appear the struct, so new commit has remove it and add key_points to record the `important point`  time used